src/browser.js

```diff
@@ -281,9 +288,9 @@ fetch('../package.json')
   .then(res => res.json())
   .then(res => {
     version = res.version;
-    const item = localStorage.getItem('searchurl');
-    if (item) {
-      createTab('https://ninetails.cf/?v=' + version + '&e=' + item);
+    const homepageValue = localStorage.getItem('homepage');
+    if (homepageValue) {
+      createTab(homepageValue);
     } else {
       createTab('https://ninetails.cf/?v=' + version);
```

Comment:
The `else` should contain something like https://github.com/MystPi/ninetails/blob/586e8fbd497a7407db31aaa314f1b6ec706a5051/src/browser.js#L285-L289

Reply:
I fixed it.

Reply:
Right! Thanks! ;)